framework/customize

```diff
@@ -183,8 +194,10 @@ _customize_update () {
       return 0
     fi
 
-    if [ "$( git branch | awk '/\*/ {print $2}')" != 'master' ]; then
-      log 'custom_environments is not on master'
+    local default_branch
+    default_branch="$(_customize_get_default_branch)"
```

Comment:
instead of using the global default branch, what about checking the customized repos default branch?

Reply:
That's what this does I think, it's fetching the default branch for this repo.